services-manager/src/main/resources/application.yaml

```diff
@@ -85,10 +85,6 @@ manager:
     #     my.service.MainApplication: "INFO"                    :: ALSO http POST address /actuator/loggers/${key}, body "configuredLevel=${value}"   - classpath to change and logging level to change to
     ###
 
-eureka:
```

Comment:
See we are removing reference to eureka in the services-manager.  There are a few mentions of eureka in the documentation.  See the line above starting with "my-service" for example